src/com/lftechnology/java/training/srijan/basics/genericclass/GenericClassDemo1.java

```diff
@@ -0,0 +1,20 @@
+package com.lftechnology.java.training.srijan.basics.genericclass;
+
+public class GenericClassDemo1 {
+	public static <T extends Comparable<T>> T maximum (T x, T y, T z){
+		T max = x;
+		if(y.compareTo(max)>0){
+			max = y;
+		}
+		if(z.compareTo(max)>0){
+			max = z;
+		}
+		return max;
+	}
+	
+	public static void main(String[] args) {
+		System.out.printf("Maximum of %d %d %d is %d\n",6,5,7,maximum(6,5,7));
+		System.out.printf("Maximum of %f %f %f is %f\n",6.6,8.8,7.7,maximum(6.6,8.8,7.7));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)